src/scripts/tweaks/show_full_note_count.js

```diff
@@ -0,0 +1,48 @@
+import { pageModifications } from '../../util/mutations.js';
+import { translate } from '../../util/language_data.js';
+import { buildStyle, postSelector } from '../../util/interface.js';
+import { timelineObject } from '../../util/react_props.js';
+import { keyToCss } from '../../util/css_map.js';
+
+const createNumberFormat = async () => {
+  const locale = document.documentElement.lang;
+  return new Intl.NumberFormat(locale);
+};
+
+const numberFormat = await createNumberFormat().catch(() => new Intl.NumberFormat());
```

Comment:
I don't believe this can fail now, so
```suggestion
const { lang } = document.documentElement;
const numberFormat = new Intl.NumberFormat(lang);
```